autovalet_husky/launch/aruco_hardware.launch

```diff
@@ -4,9 +4,9 @@
 <launch>
 
     <arg name="markerId"        default="582"/>
-    <arg name="markerSize"      default="0.255"/>    <!-- in m -->
+    <arg name="markerSize"      default="0.405"/>    <!-- in m -->
     <arg name="eye"             default="left"/>
-    <arg name="marker_frame"    default="aruco_marker_frame"/>
+    <arg name="marker_frame"    default="parking_spot"/>
```

Comment:
I don't remember if this frame id is consistent with respect to our hw parking code. You sure this would work? <Seems like a big change>

Reply:
yea I checked it. we did make the effort to bring both hardware and sim the same marker_frame in the end. This should be fine, and this was from our latest code in this testing branch, so it was working back then.

Reply:
check autovalet_system.py line 108

Reply:
I see a comment on line 109 that asks dev to check the aruco file just in case. I'm happy with that anyway even if this causes an error. 108 is parking spot so that's consistent. Thanks for checking Eva!